CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ option(FLB_IN_EVENT_TEST               "Enable Events test plugin"
 option(FLB_IN_EVENT_TYPE               "Enable event type plugin"                     Yes)
 option(FLB_IN_FLUENTBIT_METRICS        "Enable Fluent Bit metrics  plugin"            Yes)
 option(FLB_IN_FORWARD                  "Enable Forward input plugin"                  Yes)
+option(FLB_IN_GELF                     "Enable GELF input plugin"                     Yes)
```

Comment:
One thing: does this build on Mac and Windows targets? If not it needs to be disabled for those in the custom cmake config they have.